apps/obs-web/src/views/DebrieferWcgopSearch.vue

```diff
@@ -1,182 +1,127 @@
-    />
-    <debriefer-select-comp
-      label="Status"
-      style="display: inline-block"
-      class="q-mx-xs"
-      :val.sync="status"
-      lookupView="all_doc_types"
-      lookupLabel="doc.description"
-      lookupValue="id"
-      :multiple="true"
-      :lookupQueryOptions="{ key: 'trip-status', include_docs: true }"
-      @select="select('wcgop_trips_by_statusId', 'tripStatus._id', status)"
-    />
-    <debriefer-select-comp
-      label="Vessel Name"
-      style="display: inline-block"
-      class="q-mx-xs"
-      :val.sync="vessel"
-      lookupView="all_vessel_names"
-      lookupLabel="value"
-      lookupValue="id"
-      :lookupQueryOptions="{}"
-      :multiple="true"
-      @select="select('wcgop_trips_by_vesselId', 'vessel._id', vessel)"
-    />
-    <debriefer-select-comp
-      label="Fishery"
-      style="display: inline-block"
-      class="q-mx-xs"
-      :val.sync="fishery"
-      lookupView="all_doc_types"
-      lookupLabel="doc.description"
-      lookupValue="id"
-      :lookupQueryOptions="{ key: 'fishery', include_docs: true }"
-      :multiple="true"
-      @select="select('wcgop_trips_by_fisheryId', 'fishery._id', fishery)"
-    />
-    <debriefer-select-comp
-      style="display: inline-block"
-      class="q-mx-xs"
-      label="Trip Id"
-      :val.sync="tripId"
-      lookupView="wcgop_trips_by_observerId"
-      lookupLabel="value"
-      lookupValue="id"
-      :multiple="true"
-      :lookupQueryOptions="{}"
-      @select="selectTripId"
-    />
-    <div style="display: inline-block" class="q-pa-md">
-      <div class="p-float-label q-px-xs">
-        <pCalendar
-          v-model="startDate"
-          @date-select="dateFilter"
-          @clear-click="dateFilter"
-          :showButtonBar="true"
-        />
-        <label for="endDate" style="color: #027be3">Start Date</label>
+  <div class="q-pa-xs q-gutter-xs">
+      <div v-for="(option, i) of searchOptions" :key="i">
+        <q-btn-toggle
+          v-if="i > 0"
+          v-model="evaluators[i - 1]"
+          toggle-color="black"
+          color="grey-4"
+          :options="[{label: 'and', value: 'and'}, {label: 'or', value: 'or'}]"
+          dense
+          flat
+        ></q-btn-toggle>
+        <span v-if="i > 0" class="evaluator-description">{{ evaluatorDescription(i) }}</span>
+        <attribute-search-comp :val.sync=searchOptions[i]></attribute-search-comp>
       </div>
-    </div>
-    <div style="display: inline-block" class="q-pa-md">
-      <div class="p-float-label q-px-xs">
-        <pCalendar
-          v-model="endDate"
-          @date-select="dateFilter"
-          @clear-click="dateFilter"
-          :showButtonBar="true"
-        />
-        <label for="endDate" style="color: #027be3">End Date</label>
-      </div>
-    </div>
+      <q-btn @click="addFilter" color="primary" size="md">Add Filter</q-btn>
+      <q-btn @click="resetFilters" color="primary" size="md">Reset</q-btn>
```

Comment:
Can we also clear selected values as well when reset it selected.

Reply:
yes, finally figured out how to do this

Reply:
cool sounds good